src/controllers/pods.ts

```diff
@@ -115,9 +136,30 @@ export default function (server: Hapi.Server, deps: Injector) {
     }
   }
 
-  async function extendPod (request: any, h: Hapi.ResponseToolkit) {
+  async function streamExtendPod (request: any, h: Hapi.ResponseToolkit) {
```

Comment:
I think this is the same as streamPostPod so they could both be the same function but they just execute postPod or extendPod. You could just pass that function into a common function so that we don't have the code duplicated.